sass/components/form/components/_select.scss

```diff
@@ -38,6 +45,9 @@ $component-form-select-properties: $default-component-form-select-properties !de
       appearance: none;
       background-color: getThemeProperty(backgroundColor, $component-form-select-properties);
       border: 1px solid getThemeProperty(borderColor, $component-form-select-properties);
+      color: getThemeProperty(textColorDark, $component-form-select-properties);
```

Comment:
This is set to the theme property `textColorDark`, not the select `color` property. Also, this component has a `textColor` property.